xla/hlo/experimental/auto_sharding/auto_sharding_util.cc

```diff
@@ -1671,10 +1671,16 @@ HloSharding Tile(const Shape& tensor_shape,
   for (int i = 0; i < mesh_dims.size(); ++i) {
     mesh_dims_general[i].push_back(mesh_dims[i]);
   }
-  if (device_mesh.IsIota()) {
-    return TileV2(tensor_shape, tensor_dims, mesh_dims_general, device_mesh);
-  }
-  return TileV1(tensor_shape, tensor_dims, mesh_dims_general, device_mesh);
+  return Tile(tensor_shape, tensor_dims, mesh_dims_general, device_mesh);
+}
+
+HloSharding Tile(const Shape& tensor_shape,
+                 absl::Span<const int64_t> tensor_dims,
+                 std::initializer_list<int64_t> mesh_dims,
+                 const DeviceMesh& device_mesh) {
+  return Tile(tensor_shape, tensor_dims,
+              absl::Span<const int64_t>(mesh_dims.begin(), mesh_dims.size()),
```

Comment:
```suggestion
              absl::MakeConstSpan(mesh_dims.begin(), mesh_dims.size()),
```

Reply:
Thank you for your feedback! I also noticed that absl::Span has dedicated [ctor for std::initializer_list](https://docs.ros.org/en/lunar/api/abseil_cpp/html/span_8h_source.html#l00209),  so there's no need to call `begin()` and `size()`. I've updated the code accordingly."